actors/evm/src/interpreter/instructions/log.rs

```diff
@@ -4,11 +4,26 @@ use {
     fvm_ipld_blockstore::Blockstore,
 };
 
-#[inline]
+#[cfg(debug_assertions)]
 pub fn log<'r, BS: Blockstore, RT: Runtime<BS>>(
     _state: &mut ExecutionState,
-    _platform: &'r System<'r, BS, RT>,
+    _system: &'r System<'r, BS, RT>,
     _num_topics: usize,
 ) -> Result<(), StatusCode> {
-    todo!()
+    todo!("unimplemented");
+}
+
+#[cfg(not(debug_assertions))]
```

Comment:
We won't hit this in tests, but will in production (when debug assertions are turned off).